subprojects/plugins/src/main/java/org/gradle/api/plugins/JavaBasePlugin.java

```diff
@@ -346,12 +347,18 @@ public void execute(final AbstractCompile compile) {
                 conventionMapping.map("sourceCompatibility", new Callable<Object>() {
```

Comment:
I'd love to see those callables turned into lamdas :)

Reply:
I can add a commit for that indeed.